src/witan/models/dem/ccm/mort/mortality.clj

```diff
@@ -138,8 +169,9 @@
    :witan/output-schema  {:deaths DeathsOutputSchema}
    :witan/exported? true}
   [{:keys [initial-projected-mortality-rates population-at-risk loop-year]} _]
-  {:deaths
-   (cf/project-component population-at-risk
-                         initial-projected-mortality-rates
-                         loop-year
-                         :death-rate :deaths)})
+  (let [max-yr (reduce max (ds/column initial-projected-mortality-rates :year))
+        _ (utils/property-holds? max-yr #(<= loop-year %) (str loop-year " is more than loop year"))]
```

Comment:
@sunnytownsend yes it's a assertion like when we check years are valid elsewhere